static/js/popovers.js

```diff
@@ -622,11 +566,8 @@ export function open_message_menu(message) {
         return true;
     }
 
-    const message_id = message.id;
-    toggle_actions_popover($(".selected_message .actions_hover")[0], message_id);
-    if ($current_actions_popover_elem) {
-        focus_first_action_popover_item();
-    }
+    $(".selected_message .actions_hover .zulip-icon-ellipsis-v-solid").trigger("click");
+    focus_first_action_popover_item();
     return true;
 }
```

Comment:
Let's move this to `popover_menus` in a new commit; and maybe rename it to something like `open_message_actions_menu`?